### ISSUE TITLE
Line length incorrect with Unicode characters

### ISSUE THREAD

**Issue by ShadowLNC** [AtomLinter/linter-pylama#63]
I have the `minimap` and `minimap-titles` plugins. Using this, I can run "Minimap Titles: Convert" on a line, and it creates ASCII block (title) text, as shown in the image below. It uses Unicode characters that are improperly counted by `linter-pylama`, or perhaps the "parent" `linter` plugin.

You can see the error here for a line that is only about 40 characters long (the 80-wide line is near the right edge of the image):

![image](https://cloud.githubusercontent.com/assets/6551471/23783416/5538cb8a-05ab-11e7-9210-233f2beff9a6.png)

Running `ord` on the character in question returns the base 10 integer 9608.

Versions:
Linter 1.11.23
Linter Pylama 0.8.13
Minimap 4.26.8
Minimap Titles 1.10.0
Atom 1.15.0 ia32
Windows 10 Pro, 64-bit

**Comment by JLCCI** [comment]
Parent linter (pep8) most likely so you would need to report this to them directly. I personally turn off E501 in options.

**Comment by ddaanet** [comment]
Yes. NOTABUG for linter-pylama, it's for pycodestyle, probably not accounting for unicode surrogate pairs correctly.
